components/channel_view/channel_view.tsx

```diff
@@ -13,6 +13,7 @@ import PostView from 'components/post_view';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
 import {browserHistory} from 'utils/browser_history';
+import CreatePostAdvance from 'components/create_post_advance';
```

Comment:
Frankly speaking, naming it CreatePostAdvance or AdvancedCreatePost or whatever else doesn’t matter it’s temporary due to Feature Flag eventually it will be either AdvancedTextEditor or CreatePost only as we will delete duplicate once we feel all is okay with this component, and sooner we do it the better, that’s why I wasn’t in favor of feature flag but you insisted on it so here we are.

Reply:
Yeah, I have already created a ticket for that and will add these things to the ticket, and Todo in code, I have a request. Please try to refrain from naming suggestions until it’s a blunder because that’s a subjective topic. That discussion was going on before I was born and will keep going even after I die, so let’s not waste time on that topic. Thanks for the other suggestions.